tests/test_data.yml

```diff
@@ -93,7 +93,6 @@ medium:
   invalid_usernames:
     - "$very%long{invalid}user(name)"
   taken_usernames:
-    - "MediumStaff"
     - "Medium"
```

Comment:
i think you can remove ``Medium`` username which is another point of failure.

Reply:
Then what can we test for medium? Nothing?